VideoLocker/src/main/java/org/edx/mobile/util/NetworkUtil.java

```diff
@@ -168,4 +171,29 @@ public static boolean isSocialFeatureFlagEnabled(Context context, Config config)
         return isSocialEnabled && (NetworkUtil.isConnectedWifi(context) || !NetworkUtil.isOnSocialDisabledNetwork(context, config));
 
     }
+
+    /**
+     * Checks if a user has a working wifi connection or has allowed download over and active
+     * network connection, to allow a download request
```

Comment:
This can be changed to:

> Verify that there is an active network connection on which downloading is allowed. If there is no such connection, then an appropriate message is displayed.